app/models/student_loans/employment.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module StudentLoans
+  class Employment < ApplicationRecord
+    self.table_name = "student_loans_employment"
+
+    SUBJECT_ATTRIBUTES = [
+      :biology_taught,
+      :chemistry_taught,
+      :physics_taught,
+      :computer_science_taught,
+      :languages_taught,
+    ].freeze
+
+    belongs_to :eligibility, class_name: "Eligibility"
+    belongs_to :school, class_name: "School"
+
+    validate :one_subject_must_be_selected, on: [:"subjects-taught", :submit], unless: :not_taught_eligible_subjects?
+    validates :student_loan_repayment_amount, on: [:"student-loan-amount", :submit], presence: {message: "Enter your student loan repayment amount"}
+    validates_numericality_of :student_loan_repayment_amount, message: "Enter a valid monetary amount", allow_nil: true, greater_than_or_equal_to: 0, less_than_or_equal_to: 99999
+
+    delegate :name, to: :school, prefix: true, allow_nil: true
```

Comment:
```suggestion
    delegate :name, to: :school, prefix: true
```

`school` is a required association, now, so it should never be `nil`, should it?

Reply:
I think it can be `nil` when we initialise it, before we save it. I tried removing it and got a failing spec for the claim school search.

Reply:
Ah ok. That makes sense.